packages/cli/src/executions/execution.service.ts

```diff
@@ -188,7 +188,7 @@ export class ExecutionService {
 			})) as IWorkflowBase;
 
 			if (workflowData === undefined) {
-				throw new ApplicationError(
+				throw new UnexpectedError(
```

Comment:
Could this happen if the execution or workflow gets deleted? Should it be OperationalError or UserError instead?

Reply:
In this and the above cases, we've been assuming that trying to operate on a resource that no longer exists is a `UserError`. I don't fully agree and would rather err on the side of caution, but I'll go with `UserError` to be consistent.